src/js/components/dashboard/summary/DashboardSummaries.jsx

```diff
@@ -24,15 +28,18 @@ export default class DashboardSummaries extends React.Component {
                     key={result.agency_name}
                     file={file}
                     agency={result.agency_name}
-                    submissions={result.submissions} />
+                    submissions={result.submissions}
+                    inFlight={this.props.inFlight} />
             );
         });
 
         return (
             <div className="dashboard-viz submission-info">
                 <h3 className="dashboard-viz__heading">Submission Information</h3>
                 <div>
-                    {summaries}
+                    {this.props.inFlight && <LoadingMessage />}
+                    {!this.props.inFlight && this.props.hasFailed && <ErrorMessageOverlay />}
+                    {!this.props.inFlight && summaries}
```

Comment:
We should either add `!this.props.hasFailed` or reset `summaries` in the component state of `DashboardSummaryContainer`. As-is, I _think_ this could simultaneously show `summaries` for a previous request and an error message for a subsequent request. Alternatively, `if else` blocks may be easier to read here.